geoai/extract.py

```
@@ -1129,10 +1132,11 @@ def save_masks_as_geotiff(
             output_path = os.path.splitext(raster_path)[0] + "_masks.tif"
 
         # Print parameters being used
-        logger.info(f"Processing masks with parameters:")
-        logger.info(f"- Confidence threshold: {confidence_threshold}")
-        logger.info(f"- Chip size: {chip_size}")
-        logger.info(f"- Mask threshold: {mask_threshold}")
+        if verbose:
+            logger.info(f"Processing masks with parameters:")
```

Ruff (0.15.21)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)